Source/View+SnapKit.swift

```diff
@@ -178,6 +178,58 @@ public extension View {
             objc_setAssociatedObject(self, &installedLayoutConstraintsKey, newValue, objc_AssociationPolicy.OBJC_ASSOCIATION_RETAIN_NONATOMIC)
         }
     }
+    
+    internal var snp_constraints: [Constraint] {
+        get {
+            if let constraints = objc_getAssociatedObject(self, &constraintsKey) as? [Constraint] {
+                return constraints
+            }
+            return []
+        }
+        set {
+            objc_setAssociatedObject(self, &constraintsKey, newValue, objc_AssociationPolicy.OBJC_ASSOCIATION_RETAIN_NONATOMIC)
+        }
+    }
 }
 
 private var installedLayoutConstraintsKey = ""
+private var constraintsKey = ""
+
+public extension UIView {
```

Comment:
We need to get rid of this swizzle, it's not worth the risk. Probably going to involve a hidden subview we insert and track and remove when necessary.